pkg/node/datapath.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package node
+
+import "net"
+
+var (
+	// TunnelDatapath is the object which manages the datapath for tunnels.
+	TunnelDatapath TunnelEndpointOwner
+)
+
+// TunnelEndpointOwner is any type which is able to modify tunnel endpoints
```

Comment:
Perhaps,

TunnelEndpointOwner configures connectivity for a range of IP addresses (specified by prefix) via a tunnel to a specified remote tunnel endpoint address (node IP).

Reply:
Addressed in next upload.